scripts/init.sh

```diff
@@ -264,4 +294,4 @@ upgrade-frequency-no-relay)
 
   ;;
 
-esac
+esac
```

Comment:
Side note that I think most of us have new line at end of files enabled to keep this from happening.

https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files

Reply:
<img width="887" alt="image" src="https://github.com/LibertyDSNP/frequency/assets/1135694/702953e5-6dc4-49ed-a07c-332967862fd6">